packages/rancher-cis-benchmark-crds/charts/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+appVersion: v0.0.1
+description: Helm chart for installing the CRDs used by rancher cis-operator than runs security scans on a kubernetes cluster
+name: rancher-cis-benchmark-crds
+version: 0.0.1
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/namespace: cis-operator-system
+  catalog.cattle.io/release-name: rancher-cis-benchmark-crds
+  catalog.cattle.io/provides-gvr: clusterscans.cis.cattle.io/v1
```

Comment:
does CRD chart also need `provides-gvr` labels? Would that be a self-conflict condition?

Reply:
I think UI needs this annotation to be present on the separate CRD chart  @cbron can you confim pls.

Reply:
Lets add provides to both

Reply:
done